app/views/finders/_nested_facet.html.erb

```diff
@@ -1,16 +1,16 @@
 <% add_gem_component_stylesheet("select") %>
 
 <% content = capture do %>
-    <%= render "govuk_publishing_components/components/select", {
-      id: nested_facet.key,
-      label: nested_facet.name,
-      full_width: true,
-      options: nested_facet.facet_options
-    } %>
+  <%= render "govuk_publishing_components/components/select", {
+    id: nested_facet.key,
+    label: nested_facet.name,
+    full_width: true,
+    options: nested_facet.facet_options
+  } %>
```

Comment:
Nitpick: Can this indentation change be fixed up to the commit where the code was introduced?